go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/onsi/ginkgo v1.15.0
 	github.com/onsi/gomega v1.10.5
 	github.com/robfig/cron v1.2.0 // indirect
+	github.com/sirupsen/logrus v1.6.0 // indirect
```

Comment:
@junan Do you use this `logrus`?

Reply:
@Lahphim  It automatically being added once I run `go mod tidy`. I think some internal package has dependency on it